frontend/src/config.json

```diff
@@ -0,0 +1 @@
+{ "encodeDaoCoreAddress": "0x0B306BF915C4d645ff596e518fAf3F9669b97016" }
```

Comment:
Will this always be the address?

Reply:
No, it has to be changed every time it is tested locally :P If you want to try it follow the steps in the frontend/readme.md